tests/e2e/test_e2e_jobs.py

```diff
@@ -669,7 +670,10 @@ def test_job_run(helper: Helper) -> None:
 ) -> AsyncIterator[str]:
     image = await generate_image(docker)
     yield image
-    await docker.images.delete(image, force=True)
+    try:
```

Comment:
While you are here, do you mind to add `try`/`finally` around `yield image`? I was going to do this in a separate PR.

Reply:
I've double-checked recently: `pytest` executes fixutre's tear-down part even if the test raises an exception (doesn't matter is it `AssertionError` or another exception type).
No need to wrap `yield` into `try/finally` block, fixture executor behaves slightly different than `contextlib.[async]contextmanager`.